estatioapp/module/capex/dom/src/main/java/org/estatio/capex/dom/coda/CodaElement.java

```diff
@@ -39,4 +46,12 @@ public String title() {
     @Getter @Setter @Column(allowsNull = "false", length = 80)
     private String name;
 
+    @Action(semantics = SemanticsOf.SAFE)
+    @ActionLayout(contributed = Contributed.AS_ASSOCIATION)
```

Comment:
These @Action and @ActionLayout should be removed.
A @CollectionLayout(defaultView="table") should be added.